src/main/java/org/jabref/gui/edit/automaticfiededitor/twofields/TwoFieldsViewModel.java

```diff
@@ -76,12 +74,14 @@ public void copyValue() {
             String toFieldValue = entry.getField(toField.get()).orElse("");
 
             if (overwriteNonEmptyFields.get() || StringUtil.isBlank(toFieldValue)) {
-                entry.setField(toField.get(), fromFieldValue);
+                if (StringUtil.isNotBlank(fromFieldValue)) {
+                    entry.setField(toField.get(), fromFieldValue);
+                }
 
                 copyFieldValueEdit.addEdit(new UndoableFieldChange(entry,
-                        toField.get(),
-                        toFieldValue,
-                        fromFieldValue));
+                                                                   toField.get(),
+                                                                   toFieldValue,
+                                                                   fromFieldValue));
```

Comment:
sthg wrong with style config?

Reply:
I remember playing around with the code style settings on IntelliJ *(To make catch statements go to newline)*. It was on a different project. I thought it was bound to the project, but apparently not.